http/src/main/resources/init-resources/startup.sh

```diff
@@ -86,7 +105,6 @@ function failScriptIfError() {
 
 function validateCert() {
   certFileDirectory=$1
-  dockerCompose=$3
```

Comment:
pass the cmd as an argument doesn't really work. Hence moving the function down and referencing `DOCKER_COMPOSE` directly